.env.example

```
@@ -1,58 +1,79 @@
-# MUSICBRAINZ_CONTACT=you@example.com
-# MUSICBRAINZ_CONTACT_URL=https://your-site.example
-# MUSICBRAINZ_SIMILAR=on
+###########################
+# .env file used for local development (and as a template for required keys).
+# In production (e.g. GitHub Actions deploy), equivalent values come from CI secrets / server env.
+###########################
+
+# Discord Bot Credentials
+BOT_TOKEN=
+CLIENT_ID=
+GUILD_ID= # Optional: For development/testing guild commands
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)